lib/ios/RNNBackButtonOptions.m

```diff
@@ -11,11 +11,44 @@ - (void)applyOn:(UIViewController *)viewController {
 		
 		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithImage:image style:UIBarButtonItemStylePlain target:nil action:nil];
 		viewController.navigationItem.backBarButtonItem = backItem;
-	} else if (self.title) {
-		UIBarButtonItem *backItem = [[UIBarButtonItem alloc] initWithTitle:self.title
+	} else {
+		NSString *title;
+
+		if(self.title) {
+			title = self.title;
```

Comment:
You could use ternary operator here like this to make this shorter and more clear: `NSString *title = self.title ? self.title : viewController.navigationItem.title;`